recipe/meta.yaml

```diff
@@ -21,16 +21,16 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=42
     - wheel
     # These are also needed at build time.
     - bleach
-    - bokeh 2.4.3
+    - bokeh >=2.4.3,<2.5.0
     - nodejs 
     - pyct
     - packaging
-    - param
-    - pyviz_comms
+    - param >=1.9.2
+    - pyviz_comms >=0.7.4
```

Comment:
Preferably do not use pinning `>=` for host dependencies. Or the exact package version like for `bokeh` or in some cases with the upper bound `<`

Reply:
To clarify, you're recommending that I should omit lower bounds (`>=`) for host dependencies as a general rule, right? My impression was that it generally couldn't hurt to include them, but I'm likely missing something. Just for my information, could I ask what the reasoning to omit them is?

Reply:
Here are requirements for the host section https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/Reference/Recipe_standards.md#the-requirements-section:
`Version pinnings must have exact versions; comparison operators must not be used.`